usecases/no2-monitoring/index.md

```diff
@@ -0,0 +1,68 @@
+# NO₂ monitoring
+
+TROPOMI is the name of a sensor on board of the Sentinel-5 Precursor (S5P) satellite, developed to monitor atmospheric chemistry.
```

Comment:
What does TROPOMI stand for?

Reply:
Maybe also link the S5P website for more info.

Reply:
Looks like it is TROPOspheric Monitoring Instrument